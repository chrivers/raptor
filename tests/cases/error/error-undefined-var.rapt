# line 1
$ for x in missing
RUN echo {{x}}
$ endfor
# line 3