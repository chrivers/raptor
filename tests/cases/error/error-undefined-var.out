error: undefined value
 --> tests/cases/error/error-undefined-var.rapt:1:9
  |
1 | # line 1
2 | $ for x in missing
  | ^^^^^^^^^^^^^^^^^^ undefined value (in tests/cases/error/error-undefined-var.rapt:2)
3 | RUN echo {{x}}
4 | $ endfor
5 | # line 3
  |
